test/jdk/com/sun/jdi/connect/spi/GeneratedConnectors.java

```diff
@@ -25,7 +25,7 @@
  * @bug 4287596
  * @summary Unit test for "Pluggable Connectors and Transports" feature.
  *
- * When a transport service is deployed the virtual machine machine
+ * When a transport service is deployed the virtual machine
```

Comment:
```suggestion
 * When a transport service is deployed, the virtual machine
```
Let's add a comma for clarity.